Source/Assets/PlayFabEditorExtensions/Editor/Scripts/PlayFabEditorSDK/PlayFabEditorHttp.cs

```diff
@@ -52,11 +52,11 @@ private static void WriteResultFile(string url, Action<string> resultCallback, b
 
             if (api.Contains("/Server/") || api.Contains("/Admin/"))
             {
-                if (PlayFabEditorDataService.ActiveTitle == null)
-                    throw new Exception("Title not found, please select your title in the PlayFab EdEx Panel, Setting Tabs");
-
-                if (string.IsNullOrEmpty(PlayFabEditorDataService.ActiveTitle.SecretKey))
-                    throw new Exception("Title Secret key not found, please clear and re-select your title in the PlayFab EdEx Panel, Setting Tabs");
+                if (PlayFabEditorDataService.ActiveTitle == null || string.IsNullOrEmpty(PlayFabEditorDataService.ActiveTitle.SecretKey))
```

Comment:
> if (PlayFabEditorDataService.ActiveTitle == null || string.IsNullOrEmpty(PlayFabEditorDataService.ActiveTitle.SecretKey)) [](start = 15, length = 122)

Btw this can probably be replaced with a more modern C#:

 if (string.IsNullOrEmpty(PlayFabEditorDataService.ActiveTitle?.SecretKey))

Reply:
Unity doesn't support the ?. operator.